app/lib/onboarding/pages/page.dart

```diff
@@ -0,0 +1,267 @@
+import 'package:url_launcher/url_launcher.dart';
+
+import '../../../common/models/metadata.dart';
+import '../../../common/module.dart' hide MetaData;
+
```

Comment:
I would change the file name to onboarding.dart, page.dart is also annoying when you are searching for the page with ctr + p

Reply:
just for the consistency im reverting this change :P